chain/pkg/owasm/res/beeb.wat

```diff
@@ -0,0 +1,31 @@
+(module
```

Comment:
A simple test oracle script wasm code that.
- "prepare": Requests external data from sources 1, 2, 3 with call data "beeb"
- "resolve": Resolves with result "beeb"